public/js/src/game/ui/HanabiCard.js

```diff
@@ -302,6 +303,10 @@ class HanabiCard extends graphics.Group {
             // We add one because the "clue" action comes before the "turn" action
             this.turnClued = globals.turn + 1;
```

Comment:
Does anything else use the old `turnClued`?

If not, you should probably get rid of it.

Reply:
Nothing else does. I deleted it locally, but that line deletion didn't register when pushing to my fork.

Reply:
Something to be addressed, yes.

What is going on with this branch? Did you rebase a bunch of commits on master that were done by Zamiel?

Reply:
Yes, I did git pull upstream master, I think.

Reply:
That shouldn't rebase Zamiel's commits.